pkg/operator/ceph/cluster/mon/health.go

```diff
@@ -350,6 +350,36 @@ func (c *Cluster) checkHealth(ctx context.Context) error {
 		}
 	}
 
+	if c.spec.Network.MultiClusterService.Enabled {
```

Comment:
Let's only consider converting the mons if all mons are healthy
```suggestion
	if allMonsInQuorum && c.spec.Network.MultiClusterService.Enabled {
```